src/arrayMethodSort.js

```diff
@@ -4,9 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = sorted) {
+    let done = false;
+
+    while (!done) {
+      done = true;
+
+      for (let i = 1; i < this.length; i++) {
+        const first = this[i - 1];
+        const second = this[i];
+
+        if (compareFunction(first, second) > 0) {
+          done = false;
+          this[i - 1] = second;
+          this[i] = first;
+        }
+      }
+    }
+
+    return this;
   };
 }
 
+const sorted = (a, b) => {
```

Comment:
function name should contain some action
it's not clear for now what's the main action of this function